package.json

```diff
@@ -24,13 +24,15 @@
     "bootstrap": "^4.5.3",
     "i18next": "^19.8.3",
     "i18next-browser-languagedetector": "^6.0.1",
+    "lodash-es": "^4.17.15",
```

Comment:
We're importing `lodash-es` but using `lodash` in the codebase?

Also, in the Vue version, we ended up using `lodash` because this complicated things with Jest (needing to add specific configuration to transpile `lodash-es` for the tests). There might be a nice approach to handle this, but just a heads up here.